homeassistant/components/lyric/manifest.json

```diff
@@ -22,5 +22,5 @@
   "iot_class": "cloud_polling",
   "loggers": ["aiolyric"],
   "quality_scale": "silver",
-  "requirements": ["aiolyric==1.1.0"]
+  "requirements": ["aiolyric==1.1.1"]
```

Comment:
Does the dependency bump need to happen together with adding the room entities, or can it be separated?

Reply:
I submitted a patch to that library that is necessary for this change to work, so the dependency bump could be done separately, but it would have to be submitted first.

Happy to do that if you think that's the right approach and you're willing to stamp that PR for me!